ci-operator/config/openshift-eng/agent-qe-infra/tools/generate_cron_bm_abi.py

```
@@ -0,0 +1,197 @@
+
+                        new_line = f"{indent}cron: {chosen_cron}"
+                        if new_line != line:
+                            line = new_line
+                            file_modified = True
+
+                updated_lines.append(line)
+
+            if file_modified:
+                try:
+                    yaml_file.write_text("\n".join(updated_lines) + "\n")
+                except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)